samples/display/cfb_custom_font/CMakeLists.txt

```diff
@@ -9,4 +9,8 @@ project(cfb_custom_font)
 target_sources(app PRIVATE src/main.c)
 
 set(gen_dir ${ZEPHYR_BINARY_DIR}/include/generated/)
+if(CONFIG_DISPLAY_MCUX_ELCDIF)
+generate_cfb_font_for_target(app fonts/8x8text_darkGrayNoShadow.png ${gen_dir}/cfb_font_dice.h 8 8 --first 49 --last 54)
```

Comment:
Why is 8x8text_darkGrayNoShadow.png inserted here, what advantages does it bring compared to dice.png?

No word about it in commit message. Really, why was it accepted?

Reply:
the dice png is very urgly to see in 8x8 format. and I find a nice one so I put it here, the license is free font.

Reply:
To really understand the problem, what is it for a display? Which graphical display can be more limited than an EPD? Where does the limit comes, is the vertical resolution only 8 pixel?

Reply:
this is typical for a LCD display, and use a small font has many benifits, 
1. this reuqires less storage size.
2. 8 pixel is select because it is easy to get aligned with byte length and height. 
I try to parase the dice 32x32 font, and for 1 the result is:
```
        /* 49 (1) */
        {
                0x00,0x00,0x00,0x00, /* 0000 0000 0000 0000 0000 0000 0000 0000*/
                0xc0,0xff,0xff,0x03, /* 1100 0000 1111 1111 1111 1111 0000 0011*/
                0xf0,0xff,0xff,0x0f, /* 1111 0000 1111 1111 1111 1111 0000 1111*/
                0xf8,0xff,0xff,0x1f, /* 1111 1000 1111 1111 1111 1111 0001 1111*/
                0xfc,0xff,0xff,0x3f, /* 1111 1100 1111 1111 1111 1111 0001 1111*/
                0xfc,0xff,0xff,0x3f, /* 1111 1100 1111 1111 1111 1111 0011 1111*/
                0xfe,0xff,0xff,0x7f, /* 1111 1110 1111 1111 1111 1111 0111 1111*/
                0xfe,0xff,0xff,0x7f, /* 1111 1110 1111 1111 1111 1111 0111 1111*/
                0xfe,0xff,0xff,0x7f, /* 1111 1110 1111 1111 1111 1111 0111 1111*/
                0xfe,0xff,0xff,0x7f, /* 1111 1110 1111 1111 1111 1111 0111 1111*/
                0xfe,0xff,0xff,0x7f, /* 1111 1110 1111 1111 1111 1111 0111 1111*/
                0xfe,0xff,0xff,0x7f, /* 1111 1110 1111 1111 1111 1111 0111 1111*/
                0xfe,0xff,0xff,0x7f, /* 1111 1110 1111 1111 1111 1111 0111 1111*/
                0xfe,0x3f,0xfc,0x7f, /* 1111 1110 0011 1111 1111 1100 0111 1111*/
                0xfe,0x1f,0xf8,0x7f, /* 1111 1110 0001 1111 1111 1000 0111 1111*/
                0xfe,0x1f,0xf8,0x7f, /* 1111 1110 0001 1111 1111 1000 0111 1111*/
                0xfe,0x1f,0xf8,0x7f, /* 1111 1110 0001 1111 1111 1000 0111 1111*/
                0xfe,0x1f,0xf8,0x7f, /* 1111 1110 0001 1111 1111 1000 0111 1111*/
                0xfe,0x3f,0xfc,0x7f, /* 1111 1110 0011 1111 1111 1100 0111 1111*/
                0xfe,0xff,0xff,0x7f, /* 1111 1110 1111 1111 1111 1111 0111 1111*/
                0xfe,0xff,0xff,0x7f, /* 1111 1110 1111 1111 1111 1111 0111 1111*/
                0xfe,0xff,0xff,0x7f, /* 1111 1110 1111 1111 1111 1111 0111 1111*/
                0xfe,0xff,0xff,0x7f, /* 1111 1110 1111 1111 1111 1111 0111 1111*/
                0xfe,0xff,0xff,0x7f, /* 1111 1110 1111 1111 1111 1111 0111 1111*/
                0xfe,0xff,0xff,0x7f, /* 1111 1110 1111 1111 1111 1111 0111 1111*/
                0xfe,0xff,0xff,0x7f, /* 1111 1110 1111 1111 1111 1111 0111 1111*/
                0xfc,0xff,0xff,0x3f, /* 1111 1100 1111 1111 1111 1111 0011 1111*/
                0xfc,0xff,0xff,0x3f, /* 1111 1100 1111 1111 1111 1111 0011 1111*/
                0xf8,0xff,0xff,0x1f, /* 1111 1000 1111 1111 1111 1111 0001 1111*/
                0xf0,0xff,0xff,0x0f, /* 1111 0000 1111 1111 1111 1111 0000 1111*/
                0xc0,0xff,0xff,0x03,/* 1100 0000 1111 1111 1111 1111 0000 0011*/
                0x00,0x00,0x00,0x00, /* 0000 0000 0000 0000 0000 0000 0000 0000*/
        },
```

Reply:
@jfischer-phytec-iot please check my reply.